### DISCUSSION TITLE
How to disable usage of field "published"?

### DISCUSSION THREAD

**Discussion by kamuzon** [area17/twill#1111] [Q&A]
I turn it off in migrations:
createDefaultTableFields($table, true, **false**);

Where in another places (controllers, views, or...) need I to turn it totally off for this module?
Please help.

Now I get SQL error:
Column not found: 1054 Unknown column 'published' in 'where clause' (SQL: select count(*) as aggregate from `customers` where `published` = 1 and `customers`.`deleted_at` is null)

**Comment by pboivin** [comment]
Hi @kamuzon, I believe 2 small changes are needed for this to work:
- Set  `protected $indexOptions = ['publish' => false];` in your module's controller
- Remove `published` from your model's `$fillable`
